blacklist/blacklist/Scenes/Home/HomeViewController.swift

```diff
@@ -8,24 +8,172 @@
 
 import UIKit
 
-class HomeViewController: UIViewController {
-    fileprivate let presenter = HomePresenter()
+class HomeViewController: UIViewController, ViewControllerProtocol {
+    typealias P = HomePresenter
+    typealias R = HomeRouter
+
+    var presenter: HomePresenter!
+
+    struct Constants {
+        static let upcomingTableViewCell = "UpcomingTableViewCell"
```

Comment:
Shouldn't be better if we move this static constant to the UpcomingTableViewCell.swift, since it will be the same identifier for all the cell of this type.

Reply:
You are right, is better have those identifiers in the file that owns it.